### DISCUSSION TITLE
posts/2021-12-26-nextjs

### DISCUSSION THREAD

**Discussion by giscus** [cmdcolin/cmdcolin.github.io#9] [General]
# posts/2021-12-26-nextjs

Blogging for the future

https://cmdcolin.github.io/posts/2021-12-26-nextjs

**Comment by kmfb** [comment]
So I subscribe your RSS source.

**Reply by cmdcolin** [reply]
cheers :) thanks for subscribing. i don't do any analytics so always happy to get comments
